src/index.ts

```diff
@@ -1,9 +1,16 @@
-import { Client } from "discord.js";
-import { options, token } from "../config.js";
+import { Client, Intents } from "discord.js";
 import { startHandler } from "./internals/commandHandler.js";
+import { token } from "../config.json";
 
-const client = new Client(options);
+const client = new Client({
+  intents: [
+    Intents.FLAGS.GUILDS,
+    Intents.FLAGS.GUILD_EMOJIS,
+    Intents.FLAGS.GUILD_MEMBERS,
+    Intents.FLAGS.GUILD_MESSAGES,
+    Intents.FLAGS.DIRECT_MESSAGES
```

Comment:
are we gonna use guild emojis? dms?

Reply:
I'm fairly sure that we dont need any of these